ui5conblr/index.html

```diff
@@ -44,21 +42,22 @@
 					<div class="b-font size_18 color_yellow"><a href="material2018.html">SESSIONS 2018</a></div>
 				</div>
 				<div class="b-list__item">
-					<div class="b-font size_18 color_yellow top1"><a target="_blank" href="https://openui5.org/ui5con/">GERMANY</a></div>
+					<div class="b-font size_18 color_yellow"><a href="https://openui5.org/ui5con/">GERMANY</a></div>
 				</div>
 			</div>
 		</div>
-
-		<div class="b-header__logo" ></div>
+		<div class="b-header__logo" onclick="location.href='index.html'"></div>
 		<div class="b-header__delimiter"></div>
 		<div class="b-ui align_center">
-			<div class="b-font size_32 color_yellow weight_light">Thanks to all speakers and participants at UI5con Bangalore 2018 for this great event.</div>
-		</div>
-
-		<div class="b-ui align_center">
-			<p class="b-font size_18" style="color: white; margin-bottom:6px; font-size:25px"></p>
-			<p class="b-font size_18" style="color: white"></p>
+			<div class="b-font size_32 color_yellow weight_light">
+				<div class="b-font size_32 color_yellow weight_light">SAVE THE DATE</div>
+				<b>JULY 4TH 2019</b>
+				<div class="b-font size_14">Tickets available mid-June</div>
```

Comment:
Mid June? And that's enough when the event is three weeks later? Nobody needs to book any travel?